src/platform/device.gni

```diff
@@ -56,6 +56,8 @@ declare_args() {
 
 declare_args() {
   chip_openthread_ftd = chip_enable_openthread
+  chip_use_ot_lib = false
+  chip_ot_lib_source_set = ""
```

Comment:
Please document.

Please do not abbreviate "openthread".

How about just "chip_openthread_target" like we have with "mbedtls_target". It doesn't matter if it's a source_set or not.

Reply:
Sure. I will rename and document